snow/networking/handler/handler.go

```diff
@@ -414,7 +414,7 @@ func (h *handler) dispatchChans(ctx context.Context) {
 			h.StopWithError(ctx, fmt.Errorf(
 				"%w while processing async message: %s",
```

Comment:
This was the log that needed to be changed
```suggestion
				"%w while processing chan message: %s",
```

Reply:
Ok cool, I can switch that up